boards/arm/stm32f429i_disc1/stm32f429i_disc1.dts

```diff
@@ -115,6 +115,17 @@
 	pinctrl-names = "default";
 	status = "okay";
 	clock-frequency = <I2C_BITRATE_FAST>;
+
+	stmpe811@41 {
+		compatible = "st,stmpe811";
+		status = "okay";
+		reg = <0x41>;
+		int-gpios = <&gpioa 15 GPIO_ACTIVE_LOW>;
+
```

Comment:
Would it make sense to have some defaults for the board? Won't expect those to be spot on but probably good enough to hit the hello world button in the sample.

Reply:
We added some default values for dts.